models/public/brain-tumor-segmentation-0002/model.yml

```diff
@@ -0,0 +1,31 @@
+# Copyright (c) 2019 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+description: >-
+  This model was created for participation in the Brain Tumor Segmentation Challenge
+  <https://www.med.upenn.edu/cbica/brats2019/registration.html> (BraTS) 2019.
+  It has UNet architecture trained with residual blocks.
+task_type: semantic_segmentation
+files:
+  - name: brain-tumor-segmentation-0002.onnx
+    size: 21225
```

Comment:
This size is obviously incorrect. The file is many megabytes in size.

Reply:
Ok, I'll fix it. why there is no any check on downloaded size?

Reply:
Fixed

Reply:
> why there is no any check on downloaded size?

There is now. :-)